### DISCUSSION TITLE
Sending email reports from a custom domain name

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#563] [Anything Else]
"For sending the weekly reports is it possible to send it from our domain instead of using your domain name when using cloud?"

(request via email)

**Comment by ukutaht** [comment]
I think this makes a lot of sense if you can also access the dashboard from that custom domain

**Reply by metmarkosaric** [reply]
that's another frequent request. being able to share your public stats dashboard from your own custom domain
